### ISSUE TITLE
Unable to send whisper

### ISSUE THREAD

**Issue by mtilda** [mxkay/techtrans-bot#31]
Looks like I am unable to send whispers due to a permission issue. I am reading online that other people have experienced similar problems when their bot is not "verified".

Read this: https://discuss.dev.twitch.tv/t/your-setting-prevent-you-from-sending-this-whisper/26849

I submitted a request for verification, but I have a feeling it will take multiple days to resolve this issue (it seems like this bot's account might not meet all the requirements for verification).

Invoke this function to attempt a whisper (I recommend invoking this after `Connect()` and `JoinChannel()`, inside `Start()`):
```go
// Make the bot send a whisper to a user
func (bot *Bot) Whisper(message string, recipient string) error {
	_, err := bot.connection.Write([]byte("PRIVMSG #" + bot.Channel + " :/w " + recipient + " " + message + "\r\n"))
	if nil != err {
		return err
	}

	return nil
}
```

This is the response I get from the server:
```
:tmi.twitch.tv NOTICE #<CHANNEL NAME> :Your settings prevent you from sending this whisper.
```

**Comment by mtilda** [comment]
So I don't sink too much more time into this problem, I am going to put sending whispers (#6) on the backburner until I hear back.

**Comment by nicholascassidy** [comment]
Good catch! Yep - they changed their policy. As long as you can talk through your code, that'll be enough.

**Comment by mtilda** [comment]
Great! Thank you again :)
